src/encoded/regulome_indexer.py

```diff
@@ -1139,7 +1180,7 @@ def add_file_to_regions_es(self, request, afile, file_doc, snp=False):
                         continue  # Skip for 63 invalid peak in a non-ENCODE ChIP-seq result, exo_HelaS3.CTCF.bed.gz
                     if chrom not in SUPPORTED_CHROMOSOMES:
                         continue   # TEMPORARY: limit both SNPs and regions to major chroms
-                    if chrom not in file_data:
+                    if (chrom not in file_data) or (len(file_data[chrom]) > 3e6):
```

Comment:
This seems a little arbitrary, what is going on here?  3,000,000?

Reply:
It is arbitrary. If we hold all SNP docs of one chromosome in memory before putting into index, it will go out of memory. Any suggestions on better strategies here?

Reply:
at least define a constant like "MAX_SNP_FILE = 3e6",  better would be to stream the file using IOStream.  Not sure this is worth editing.